<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSMConverterDialog</class>
 <widget class="QDialog" name="OSMConverterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>208</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>OSM Converter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="uploadLabel">
     <property name="text">
      <string>OSM Input File:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="OSMConverterLineEdit" name="mUploadLineEdit">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string extracomment="The input *.osm file"/>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Drag and drop or select a .osm file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mUploadButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>AFSIM Output File:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="OSMConverterLineEdit" name="mTargetLineEdit">
       <property name="toolTip">
        <string extracomment="The path for the plugin output"/>
       </property>
       <property name="placeholderText">
        <string>Provide a path for the plugin output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mTargetButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>AFSIM route_network &lt;name&gt;:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="mNameLineEdit">
     <property name="toolTip">
      <string extracomment="The route_network name"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Provide the route_network name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="mWayTagButton">
       <property name="text">
        <string>Way Tag Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mNodeTagButton">
       <property name="text">
        <string>Node Tag Data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mExportButton">
       <property name="toolTip">
        <string extracomment="Convert the OSM format to route_network format"/>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OSMConverterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../source/OSMConverterCustomWidgets.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mUploadLineEdit</tabstop>
  <tabstop>mUploadButton</tabstop>
  <tabstop>mTargetLineEdit</tabstop>
  <tabstop>mTargetButton</tabstop>
  <tabstop>mNameLineEdit</tabstop>
  <tabstop>mWayTagButton</tabstop>
  <tabstop>mNodeTagButton</tabstop>
  <tabstop>mExportButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
